Makefile

```diff
@@ -52,8 +54,17 @@ trusted_client.riscv: $(TCLIENT)
 $(OBJS): %.o: %.cpp
 	$(CC) $(CCFLAGS) -c $<
 
+getandsethash:
+	./scripts/get_attestation.sh ./include
+
+copysdk copysdk1:
+	yes | cp -rf *.riscv server_eapp/server_eapp.eapp_riscv $(KEYSTONE_SDK_DIR)/bin/
```

Comment:
does `cp *.riscv server_eapp/server_eapp.eapp_riscv $(KEYSTONE_SDK_DIR)/bin/*` work instead? (or prefix with `-` if we don't care about success)

Reply:
I forgot to change that back to just cp. I was having issues and realized it was just not rebuilding it when the hash was changed and originally though it may have been an issue with not overriding the file but realized it was just not tracked. I will change this back.